src/Avalonia.Themes.Fluent/FluentTheme.cs

```diff
@@ -58,6 +64,18 @@ public FluentThemeMode Mode
             get => GetValue(ModeProperty);
             set => SetValue(ModeProperty, value);
         }
+        
+        public static readonly StyledProperty<DensityStyle> DensityStyleProperty =
+            AvaloniaProperty.Register<FluentTheme, DensityStyle>(nameof(DensityStyle));
+        /// <summary>
```

Comment:
also i think it makes sense to add space between 69 and 70 lines and 58\59 lines(even though its not your code, just for beautification)